src/styles/main.scss

```diff
@@ -1,3 +1,52 @@
-body {
-  margin: 0;
+@import "./utils/variables.scss";
+@import "./utils/mixins.scss";
+@import "./reset.scss";
+
+html {
+  font-family: Arial, sans-serif;
+}
+
+@each $name, $margin-size in $days {
+  .calendar__start-day--#{$name} >:first-child {
+    margin-left: #{$margin-size};
+  }
+}
+
+@for $i from 28 to 31 {
+  .calendar__month-length--#{$i} >:nth-child(n + #{$i + 1}) {
+    display: none;
+  }
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: calc((#{$block-size} + 1px) * 7 + (#{$calendar-padding} * 2));
+  gap: 1px;
+  padding: $calendar-padding;
+  margin: auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.